systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -179,6 +187,77 @@ void testKafkaExporterDifferentSetting() throws InterruptedException, ExecutionE
+
+        connectResponse = Pattern.compile("strimzi_resources\\{kind=\"KafkaBridge\",} ([\\d.][^\\n]+)");
+        values = MetricsUtils.collectSpecificMetric(connectResponse, clusterOperatorMetricsData);
+        assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 0));
+
+        connectResponse = Pattern.compile("strimzi_resources\\{kind=\"KafkaConnect\",} ([\\d.][^\\n]+)");
+        values = MetricsUtils.collectSpecificMetric(connectResponse, clusterOperatorMetricsData);
+        assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 1));
+
+        connectResponse = Pattern.compile("strimzi_resources\\{kind=\"KafkaConnectS2I\",} ([\\d.][^\\n]+)");
+        values = MetricsUtils.collectSpecificMetric(connectResponse, clusterOperatorMetricsData);
+        assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 0));
+
+        connectResponse = Pattern.compile("strimzi_resources\\{kind=\"KafkaMirrorMaker\",} ([\\d.][^\\n]+)");
+        values = MetricsUtils.collectSpecificMetric(connectResponse, clusterOperatorMetricsData);
+        assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 0));
+
+        connectResponse = Pattern.compile("strimzi_resources\\{kind=\"KafkaMirrorMaker2\",} ([\\d.][^\\n]+)");
+        values = MetricsUtils.collectSpecificMetric(connectResponse, clusterOperatorMetricsData);
+        assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 1));
+
+        connectResponse = Pattern.compile("strimzi_resources\\{kind=\"KafkaConnector\",} ([\\d.][^\\n]+)");
+        values = MetricsUtils.collectSpecificMetric(connectResponse, clusterOperatorMetricsData);
+        assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 0));
+    }
+
+    @Test
+    void testUserOperatorMetrics() {
+        assertCoMetricNotNull("strimzi_reconciliations_locked_total", "KafkaUser");
+        assertCoMetricNotNull("strimzi_reconciliations_successful_total", "KafkaUser");
+        assertCoMetricNotNull("strimzi_reconciliations_duration_seconds_count", "KafkaUser");
+        assertCoMetricNotNull("strimzi_reconciliations_duration_seconds_sum", "KafkaUser");
+        assertCoMetricNotNull("strimzi_reconciliations_duration_seconds_max", "KafkaUser");
+        assertCoMetricNotNull("strimzi_reconciliations_periodical_total", "KafkaUser");
+        assertCoMetricNotNull("strimzi_reconciliations_failed_total", "KafkaUser");
+        assertCoMetricNotNull("strimzi_reconciliations_total", "KafkaUser");
+
+        Pattern connectResponse = Pattern.compile("strimzi_resources\\{kind=\"KafkaUser\",} ([\\d.][^\\n]+)");
+        ArrayList<Double> values = MetricsUtils.collectSpecificMetric(connectResponse, userOperatorMetricsData);
+        assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 2));
+    }
+
+    @Test
+    void testMirrorMaker2Metrics() {
+        Pattern connectResponse = Pattern.compile("kafka_connect_connect_worker_metrics_connector_count ([\\d.][^\\n]+)");
```

Comment:
Pay attention at #2800 , it's going to change `kafka_connect_connect_worker_metrics_connector_count` to `kafka_connect_worker_connector_count`